video/avi_decoder.cpp

```diff
@@ -266,9 +266,12 @@ void AVIDecoder::handleStreamHeader(uint32 size) {
 	sHeader.size = size;
 	sHeader.streamType = _fileStream->readUint32BE();
 
-	if (sHeader.streamType == ID_MIDS || sHeader.streamType == ID_TXTS)
+	if (sHeader.streamType == ID_MIDS)
 		error("Unhandled MIDI/Text stream");
 
+	if (sHeader.streamType == ID_TXTS)
```

Comment:
The video in Duckman contains unused text streams. I've changed the error to a warning to allow the videos to play.